### ISSUE TITLE
Does wrong language used in aria-label break 3.1.1?

### ISSUE THREAD

**Issue by AgataJedryszek-da** [w3c/wcag#3003]
If the page has `lang` attribute correctly defined, but there is an `aria-label` with text in different language that should be in the same language, does it break 3.1.1?

**Comment by patrickhlauke** [comment]
not 3.1.1 Language of **Page**, but i'd say 3.1.2 Language of **Parts**

**Comment by alastc** [comment]
@AgataJedryszek-da I think that's a straightforward answer, I'll close this issue. Please re-open if you disagree, or wish to propose a change to the WCAG documentation.
